sklearn/plot/_gridsearch_results.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def plot_gridsearch_results(cv_results, metric='mean_test_score',
```

Comment:
i guess 1D grid-search is basically the same thing as validation curve. maybe in [this example](http://scikit-learn.org/stable/auto_examples/model_selection/plot_validation_curve.html#sphx-glr-auto-examples-model-selection-plot-validation-curve-py), we can say that this can also be done using grid search and its a much easier code?